nat_gateway/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "private_route_tables" {
```

Comment:
Lets say I have 2 groups of private subnets, a proxy group and an app group. How will this work with the route table defined in `nat_gateway/main.tf` on line 12